opgee/utils.py

```diff
@@ -337,12 +321,10 @@ def getResource(relpath):
     import pkgutil
 
     try:
-        contents = pkgutil.get_data('opgee', relpath)
+        return pkgutil.get_data('opgee', relpath)
     except FileNotFoundError as e:
         raise OpgeeException(f"Resource '{relpath}' was not found in the opgee package: {e}")
 
-    return contents.decode('utf-8')
```

Comment:
I'm surprised this works. I added the `decode('utf-8')` at one point by necessity. I think it was returning unicode and needed decoding. However, now it looks like this is called only from `tests/test_utils.py`, so it could be deleted.